packages/mrml-python/mrml.pyi

```diff
@@ -64,12 +64,12 @@ class ParserOptions:
 
 class RenderOptions:
     """RenderOptions configures rendering behavior, including whether to disable comments and how to handle social icons and fonts."""
-    def __init__(
-        self,
-        disable_comments: bool = False,
-        social_icon_origin: Optional[str] = None,
-        fonts: Optional[Dict[str, str]] = None,
-    ) -> None: ...
+
+    disable_comments: bool
+    social_icon_origin: Optional[str]
+    fonts: Optional[Dict[str, str]]
+
+    def __init__(self) -> None: ...
```

Comment:
This type def was slightly off as RenderOptions doesn't take these properties in the constructor.